### PR TITLE
config file to build.gradle

### PR DESCRIPTION
**Pull request by pbk20191**
this will help update and building new workspace. from now on you dont need to download extra files like codechicken and nei

### CODE REVIEW

**Comment by HbmMods** [comment]
How exactly does it work? And why does it have so many maven repositories of other mods in it?

File: build.gradle [hunk 1]
```diff
@@ -1,56 +1,101 @@
...
+archivesBaseName = "hbm"
+
+minecraft {
+    version = "${config.minecraft.version}-${config.forge.version}-${config.minecraft.version}"
+    runDir = "eclipse"
+}
+repositories {
+    maven {
+        name 'Forge' // Applied Energistics
+        url 'http://files.minecraftforge.net/maven'
```
Comment (pbk20191) [inline build.gradle:58]: this contains AE2 and other mods

File: build.gradle [hunk 2]
```diff
@@ -1,56 +1,101 @@
...
+    }
+    maven {
+        name = "chickenbones"
+        url = "http://chickenbones.net/maven/"
```
Comment (pbk20191) [inline build.gradle:62]: this is for chickenbones: NEI, codechickenCore, CodechickenLib

**Comment by pbk20191** [comment]
> How exactly does it work? And why does it have so many maven repositories of other mods in it?

most of this maven are for just for future dependencies, but only needed is name = "chickenbones", you can remove other maven. I would recommand you to don't remove at least <name "Mobius Repo"> maven for Waila support.

File: build.gradle [hunk 3]
```diff
@@ -1,56 +1,101 @@
...
+
+apply plugin: 'forge'
+
+allprojects {
+    apply plugin: 'java'
+    sourceCompatibility = 1.8
+    targetCompatibility = 1.8
+}
+
+file "build.properties" withReader {
```
Comment (pbk20191) [inline build.gradle:32]: this will read "build.properties" and extract data

**Comment by pbk20191** [comment]
> How exactly does it work? And why does it have so many maven repositories of other mods in it?

before doing any gradle command, it will read build.properties file and check dependancy. and if dependancy fails than it will automatically download and complie the required files

**Comment by pbk20191** [comment]
umm, this is out of topic. Could you please move /asset into the /resource/ directory? putting asset file inside the source file can cause problem with other resource loading mods. and that is not friendly for modders

**Comment by pbk20191** [comment]
sorry.. I accidently mixed up my PR. I will remake the PR when I'm done.

**Pull request closed without merging**